src/eslint-config-adeira/ourRules.js

```diff
@@ -635,12 +626,10 @@ module.exports = {
   'eslint-comments/no-unused-enable': ERROR,
   'eslint-comments/no-use': OFF,
 
-  // Relay Flow type imports
-  'relay-imports/no-values': ERROR,
-  'relay-imports/type-must-exist': ERROR,
-
-  // Kiwi.com custom rules
+  // Adeira custom rules
   'adeira/only-nullable-fields': ERROR,
   'adeira/no-invalid-flow-annotations': ERROR,
   'adeira/no-internal-flow-type': NEXT_VERSION_ERROR, // TODO: ERROR
+  'adeira/relay-import-no-values': ERROR,
+  'adeira/relay-import-type-must-exist': ERROR,
```

Comment:
Should we keep the plural version? 🤔 Original rules were `relay-imports` with "s".

Reply:
My idea was that before, it would have been `relay-imports/no-values`, but now it is part of `adeira/` so singular sounded better in my head. 

Not really that important to me. If you want me to change it back, I am fine with it it.